plans/benchmarks/benchmarks.go

```diff
@@ -31,8 +55,10 @@ func StartTimeBench(runenv *runtime.RunEnv) error {
 	elapsed := time.Since(runenv.TestStartTime)
 	emitTime(runenv, "Time to start", elapsed)
 
-	gauge := runenv.M().NewGauge(runtime.GaugeOpts{Name: "start_time", Help: "time from plan scheduled to plan booted"})
-	gauge.Set(float64(elapsed))
+	w := metricsWriter(runenv)
+	w.WritePoint(makePoint(runenv, "time to start", map[string]interface{}{"start_time": elapsed}))
+	w.Flush()
```

Comment:
My concern with the "immediate send" approach is that:

a) we don't _really_ need the data to be available _immediately_.
b) Influxdb will have to cope with bursty writes, coming from as many as 10k clients.
c) sending data continuously can affect the performance of the test case itself, as well as the network.

Related: I don't think we should be flushing the batch on every single write.